src/app/functions.js

```diff
@@ -0,0 +1,201 @@
+import i18n from '../i18n'
+
+const sumAll = numbers => {
+  let points = 0
+  numbers.forEach(number => {
+    points = points + number
+  })
+  return points
+}
+
+export const isThreeOfKind = numbers => {
+  const sortedArray = numbers.sort((a, b) => a > b)
+  if (sortedArray.length > 5) return false
+  if (isFullHouse(numbers)) return true
```

Comment:
Why to check this? It involves a dependency to another function and isn't needed, or?